### PR TITLE
More Keyboard fixes

### PR DESCRIPTION
**Pull request by Lulonaut**
Allows for holding down keys in the Storage Overlay and Item list search bar

### CODE REVIEW

**Review by NopoTheGamer — APPROVED** [review]
github :smoking:

File: src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java [hunk 1]
```diff
@@ -1907,27 +1854,18 @@ private ItemStack getWardrobeSlot(int armourSlot) {
...
 	/**
 	 * Renders the search bar, quick commands, item selection (right), item info (left) and armor hud gui elements.
 	 */
 	public void render(boolean hoverInv) {
 		if (disabled) {
 			return;
 		}
+		if (Keyboard.isRepeatEvent()) {
+			keyboardInput(false);
+			Utils.sleepOnCurrentThread(15);
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1866]: Why are we sleeping on the rendering thread. Even for just 15ms that seems like a bad idea

Comment (Lulonaut) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1866]: Without that the characters will be removed/added really really fast and I didn't notice any issues with it

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1866]: I feel like you are slowing down the fps so we dont type too fast? At least i think thats whtat you are doing

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1866]: Wait why are we processing userinput in the render event in the first place??

**Review by NopoTheGamer — DISMISSED** [review]

**Review by DoKM — APPROVED** [review]
looksfine

**Review by NopoTheGamer — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Review by DeDiamondPro — APPROVED** [review]

**Pull request merged**